server/signalfx.go

```diff
@@ -55,6 +55,9 @@ func (s *signalfxDrain) Flush() error {
 	if err != nil {
 		panic(errors.Wrap(err, "failed to marshal payload to JSON"))
 	}
+
+	s.points = []datapoint{} // release accumulated memory
+
```

Comment:
This should probably be later in the function?  If we fail to send to signalfx, it seems we'd want to re-submit those points?